exec/java-exec/src/main/codegen/templates/CastFunctions.java

```diff
@@ -54,14 +56,14 @@ public void setup() {}
   public void eval() {
     <#if (type.from.startsWith("Float") && type.to.endsWith("Int"))>
     boolean sign = (in.value < 0);
-    in.value = java.lang.Math.abs(in.value);
-    ${type.native} fractional = in.value % 1;
+    ${type.native} value = java.lang.Math.abs(in.value);
+    ${type.native} fractional = value % 1;
```

Comment:
`out.value = Math.round(in.value)` ?

`Math.round()` rounds a double to the closest long; handling overly large and small values, which our code does not do. The Java method is likely to be much more efficient than this code.

Since the particular case is cast from `double` to `int`:

`out.value = Math.round((float) in.value)`

Returns an int.

I realize that this is existing code. But, it is broken. So, we might as well fix it the right way...